scripted/sbt/src/main/scala/sbt/test/SbtHandler.scala

```diff
@@ -80,7 +80,10 @@ final class SbtHandler(directory: File,
     val thread = new Thread() { override def run() = { p.exitValue(); server.close() } }
```

Comment:
I think the issue here is memory since all these tests are passing locally. Can we discuss migrating CI to a more powerful machine, either Drone (free) or paying for a more powerful Travis?
  
We need to find a good way to move forward.